service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillIndexAction.kt

```diff
@@ -92,53 +93,9 @@ class BackfillIndexAction @Inject constructor(
                             +backfill.state.name
                           }
                         }
-                        //                    p("mt-1 truncate text-sm text-gray-500") { +"""Regional Paradigm Technician""" }
                       }
                     }
                   }
-                  // Buttons
```

Comment:
?

Reply:
Was code copied in from Tailwind but didn't end up using.